src/ts/components/listselector.ts

```diff
@@ -10,11 +10,45 @@ export interface ListItem {
   label: string;
 }
 
+/**
+ * Filter function that can be used to filter out list items added through {@link ListSelector.addItem}.
+ *
+ * Using this does not make a lot of sense when items are added from outside because the added items can be
```

Comment:
I would remove the judgement of the usage of this interface. Just state what it does. Users should decide on their own whether to use it or not.

Reply:
This statement explains the usecase.. I will improve it